core/domain/state_domain.py

```diff
@@ -353,192 +406,1900 @@ def __init__(
     def to_dict(self):
         """Returns a dict representing this Solution domain object.
 
-        Returns:
-            dict. A dict mapping all fields of Solution instance.
-        """
-        return {
-            'answer_is_exclusive': self.answer_is_exclusive,
-            'correct_answer': self.correct_answer,
-            'explanation': self.explanation.to_dict(),
-        }
+        Returns:
```

Comment:
Do you happen to know why the diff here is so hard to read? I can't easily tell what code you've changed because github seems to regard the whole section as a big diff..

Reply:
Fixed, didn't know what cause this but it is fixed now.